lib/platform/bitbucket-server/index.js

```diff
@@ -651,6 +658,25 @@ async function createPr(
 ) {
   logger.debug(`createPr(${branchName}, title=${title})`);
   const base = useDefaultBranch ? config.defaultBranch : config.baseBranch;
+  let reviewers = [];
+
+  /* istanbul ignore else */
```

Comment:
is there an else?

Reply:
just to remove the istanbul info, that the else path is not taken.